README.md

```diff
@@ -33,18 +33,19 @@ For either platform, you can alternatively pass down a child element that will b
-| items                   | array    | _ The items for the component to render. Each item should be in the following format:<br>`{label: 'Orange',value: 'orange',key: 'orange', color: 'orange'}`<br>_ The label and the value are required, but the key will be based upon the label if it isn't included<br>\* The value can be any data type. The color is optional. | Y         | Both          |
-| placeholder             | object   | _ An override for the default placeholder object with a label of `Select an item...` and a value of `null`<br>_ An empty object can be used if you'd like to disable the placeholder entirely                                                                                                                                     | N         | Both          |
-| disabled                | boolean  | Disables interaction with the component                                                                                                                                                                                                                                                                                           | N         | Both          |
-| value                   | any      | Will attempt to locate a matching value from the `items` array by checking each item's `value` property. If found, it will update the component to show that item as selected. If the value is not found, it will default to the first item.                                                                                      | N         | Both          |
-| style                   | object   | Style overrides for most parts of the component. More details below.                                                                                                                                                                                                                                                              | N         | Both          |
-| hideDoneBar             | boolean  | Hides the bar with tabbing arrows and Done link to exit the modal. While this is typical on `select` elements on the web, the [interface guidelines](https://developer.apple.com/ios/human-interface-guidelines/controls/pickers/) does not include it.                                                                           | N         | iOS           |
-| hideIcon                | boolean  | Hides the floating downward arrow on the right side of the input box                                                                                                                                                                                                                                                              | N         | iOS           |
```

Comment:
Is it possible to just limit these changes to just the updates for `itemKey`?

Reply:
Unfortunately, no – the table formatting changed for each row, since I made one of the columns wider.